src/app/core/components/navigation/project/style.scss

```diff
@@ -12,7 +12,7 @@
 @import 'variables';
 
 .km-project-selector {
-  height: $toolbar-height - 8px / 4;
+  height: $toolbar-height - 8px * .25;
```

Comment:
What is the order of operations here? Can we replace it with `2px`?

Reply:
we definitely should :smile: